iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/DnsZoneMissingDnssecCheck.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.terraform.checks.gcp;
+
+import org.sonar.check.Rule;
+import org.sonar.iac.terraform.checks.AbstractNewResourceCheck;
+
+@Rule(key = "S6402")
+public class DnsZoneMissingDnssecCheck extends AbstractNewResourceCheck {
```

Comment:
```suggestion
public class DnsZoneCheck extends AbstractNewResourceCheck {
```
Maybe this name is a bit handier